Source/Core/CorridorGeometry.js

```diff
@@ -597,7 +597,7 @@ define([
             length = topNormals.length;
 
             var extrudeNormals = new Float32Array(length * 6);
-            for (i = 0; i < length; i ++) {
+            for (var i = 0; i < length; i ++) {
```

Comment:
Thanks for the review!

I'll open a separate issue, but there are dozens of times where a `for` loop initializes `i`, and subsequent loops in the same function (scope) use `i` again. [`block-scoped-var`](http://eslint.org/docs/rules/block-scoped-var) is the rule that would check it.